### PR TITLE
Hectorh/adding react ext

### PR DESCRIPTION
**Pull request by hectorhdzg**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: src/AITelemetry.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ReactPlugin } from '@microsoft/applicationinsights-react-js';
+
```
Comment (hiraldesai) [inline src/AITelemetry.js]: Just nitpicky but I'd call this `TelemetryService` or something so the calls below don't feel ambiguous. 

e.g. `telemetryService.AI.track...`

File: src/App.jsx [hunk 2]
```diff
@@ -1,30 +1,22 @@
 import React from 'react';
-import { ApplicationInsights, SeverityLevel } from '@microsoft/applicationinsights-web';
+import { SeverityLevel } from '@microsoft/applicationinsights-web';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```
Comment (pviotti) [inline src/App.jsx]:
```
./src/App.jsx
  Line 3:  'Router' is defined but never used  no-unused-vars
```

Comment (hectorhdzg) [reply src/App.jsx]: I created a bug related to failing tests, will take care of it later this iteration

**Comment by aruniverse** [comment]
Any update on the import problem?

**Pull request merged**